lib/navigation/routes_generator.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_localization/screens/first_screen.dart';
+import 'package:flutter_localization/screens/second_screen.dart';
+
+class RouteGenerator {
+  static Route<dynamic> generateRoute(RouteSettings settings) {
+    switch (settings.name) {
+      case '/':
```

Comment:
You can use `if else` instead of `switch` - it allows you to use constants from AppRouted